### PR TITLE
Updated file

### PR DESCRIPTION
**Pull request by mk3jamie**
**Pull Request Test 1**

Tasks completed:
- Navigate to github repo
- Locate issue #3 
- Make changes

To Do:
- Complete changes
- Close and merge

### CODE REVIEW

File: _slides/09-mk3jamie.md [hunk 1]
```diff
@@ -3,7 +3,7 @@ layout: slide
 title: "Welcome to our slide deck!"
 ---
 
-Caption here
+Here is my text update!!
```
Comment (SHills96) [inline _slides/09-mk3jamie.md:6]: Terrible but it'll have to do.

**Review by SHills96 — APPROVED** [review]
Approved.

File: _slides/09-mk3jamie.md (hunk 1 above)
Comment (SHills96) [reply _slides/09-mk3jamie.md:6]: typo. Terrific*

**Comment by mk3jamie** [comment]
Super job.

**Pull request merged**